docs/fundamentals/productivity/code-analysis.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Code analysis in .NET
+titleSuffix: ""
+description: Learn about source code analysis in .NET.
+ms.date: 08/20/2020
+ms.topic: overview
+helpviewer_keywords:
+- code analysis
+- code analyzers
+---
+# Overview of .NET source code analysis
+
+.NET code analyzers inspect your C# or Visual Basic code for security, performance, design, and other issues. Starting in .NET 5.0, these analyzers are included with the .NET SDK. You can also install third party analyzers, such as [StyleCop](https://www.nuget.org/packages/StyleCop.Analyzers/), [Roslynator](https://www.nuget.org/packages/Roslynator.Analyzers/), [XUnit Analyzers](https://www.nuget.org/packages/xunit.analyzers/), and [Sonar Analyzer](https://www.nuget.org/packages/SonarAnalyzer.CSharp/).
+
+> [!TIP]
+> In previous versions of .NET, you can install the .NET code analyzers as a [NuGet package](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers). For more information, see [Install code analyzers](/visualstudio/code-quality/install-fxcop-analyzers).
```

Comment:
Actually users can enable the .NET analyzers that ship in the box even on earlier projects by setting certain MSBuild property. The only requirement is users need to be on .NET5 SDK or later. @mikadumont is planning to write up a doc for this part. I’ll also forward you an email for your reference.